src/site/content/en/learn-web-vitals/lab-and-field-data-differences/index.md

```diff
@@ -0,0 +1,386 @@
+---
+layout: post
+title: Why lab and field data can be different (and what to do about it)
+subhead: |
+  Learn why tools that monitor Core Web Vitals metrics may report different numbers, and how to interpret those differences.
+description: |
+  Learn why tools that monitor Core Web Vitals metrics may report different numbers, and how to interpret those differences.
+authors:
+  - philipwalton
+date: 2021-08-17
+updated: 2021-08-17
+hero: image/eqprBhZUGfb8WYnumQ9ljAxRrA72/OKW9sizk0a8UloNOFx9g.jpeg
+alt: Users on their phones
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - performance
+  - web-vitals
+---
+
+Google provides [a number of tools](/vitals-tools/) to help site owners monitor
+their [Core Web Vitals](/vitals/#core-web-vitals) scores. These tools fall into
+two main categories:
+
+* Tools that report **lab data**—data collected in a controlled environment with
+  predefined device and network settings.
+* Tools that report **field data**—data collected from the real users visiting
+  your site.
+
+The problem is that sometimes the data reported by lab tools can be quite a bit
+different from the data reported by field tools! Your lab data might indicate
+that your site performs great, but your field data suggests it needs
+improvement. Alternatively, your field data may say all your pages are good, but
+your lab data may report a very low score.
+
+The following real example of a PageSpeed Insights report from web.dev shows
+that in some cases lab and field data can be different across all of the Core
+Web Vitals metrics:
+
+{% Img src="image/eqprBhZUGfb8WYnumQ9ljAxRrA72/yN4PRURHJnDaLSJKsNsW.png", alt="Screenshot of a PageSpeed Insights report with conflicting lab and field data", width="800", height="509" %}
```

Comment:
I find the annotations on this image pretty confusing. First I was confused by the shape of the arrows, I thought that the point of the intersection of each pair should be connected to a label. So the first pair could be related to "Lab data", second pair ??, third pair to "the website". Only after going through that mentally and not finding much sense, I figured out that only the pointy part of the arrow is important, so the first pair is for LCP, second CLS, and then for the third I wasn't sure if the arrow should be in between TBT and TTI or if it was off by mistake. Then I figured it's connecting both TTI and TBT to FID.

The other thing that's confusing me is why is lab LCP connected to the corresponding field bucket (yellow), good lab CLS connected to the worst field bucket (while the field data "verdict" is yellow/needs improvement), and then TBT+TTI point to the field value of FID (not the bucket like the rest). 

Here's my attempt at annotating this image:

<img width="986" alt="Screenshot 2021-08-17 at 15 56 43" src="https://user-images.githubusercontent.com/17127567/129740173-d11e5551-670e-43ef-aed1-17790f0e6a96.png">

Reply:
Great suggestion! Updated.